File: Python/Program/Komponen.py
class Komponen:
    def __init__(self, merk="", nama=""):
        self.merk = merk
        self.nama = nama

    def set_merk(self, merk):
        self.merk = merk

    def get_merk(self):
        return self.merk

    def set_nama(self, nama):
        self.nama = nama

    def get_nama(self):
        return self.nama

File: Java/Program/Komponen.java
public class Komponen {
    private String merk;
    private String nama;

    public Komponen() {
    }

    public Komponen(String merk, String nama) {
        this.merk = merk;
        this.nama = nama;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public String getMerk() {
        return this.merk;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNama() {
        return this.nama;
    }
}